thetis/options.py

```diff
@@ -124,8 +124,34 @@ class SemiImplicitTimestepperOptions3d(ExplicitTimestepperOptions3d):
         help='implicitness parameter theta for 2D solver. Value 1.0 implies fully implicit formulation.').tag(config=True)
 
 
-class GLSModelOptions(FrozenHasTraits):
-    """Options for generic length scale turbulence model"""
+class TurbulenceModelOptions(FrozenHasTraits):
+    """Abstract base class for all turbulence model options"""
+    def update(self, options):
+        if isinstance(options, dict):
+            params_dict = options
+        else:
+            # assume HasTraits
+            params_dict = options._trait_values
+        for key in params_dict:
+            self.__setattr__(key, params_dict[key])
+
+
+class PacanowskiPhilanderModelOptions(TurbulenceModelOptions):
+    """Options for Pacanowski-Philander turbulence model"""
+    max_viscosity = PositiveFloat(5e-2, help=r"float: Constant maximum viscosity :math:`\nu_{max}`").tag(config=True)
+    alpha = PositiveFloat(10.0, help="float: Richardson number multiplier").tag(config=True)
+    exponent = PositiveFloat(2.0, help=r"float: Exponent of viscosity numerator :math:`n`").tag(config=True)
+
+    def print_summary(self):
```

Comment:
Same here: we should have a method to print the contents of any of the Options classes. It would be nice to have this as a  `__str__()` method, so you can simply do:

    print(PacanowskiPhilanderModelOptions(...))